### PR TITLE
Single script run command

### PR DESCRIPTION
**Pull request by binaryberserker**
use concurrently package to run tailwind watch, rollup watch, and start express locally
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,9 +4,12 @@
   "description": "",
   "main": "server.js",
   "scripts": {
+    "dev":"concurrently --kill-others --names \" CSS \",\" JS \",\" SERVER \" -c yellow,blue,green \"npm:fe:css:build\" \"npm:fe:js:build\" \"npm:start-local\"",
```
Comment (binaryberserker) [inline package.json:7]: here's the magic line

**Comment by DavidNic11** [comment]
lgtm - there are some conflicts; can you resolve them?

File: public/dist/hosted-main.min.js
Comment (DavidNic11) [inline public/dist/hosted-main.min.js:1]: Dist shouldn't be committed any more. Can you clear you dist cache and remove it from your pr

**Review by DavidNic11 — CHANGES_REQUESTED** [review]
rm dist from the pr

**Review by DavidNic11 — APPROVED** [review]

**Pull request merged**